eslint.config.js

```diff
@@ -88,7 +88,8 @@ module.exports = [
           parser: parserTypeScriptESLint,
           parserOptions: {
             allowAutomaticSingleRunInference: true,
-            project: "./tsconfig.eslint.json",
+            project: true,
```

Comment:
Nit: I believe you should be able to remove this now.

```suggestion
```

(if not, that's a bug on us in typescript-eslint we should fix!)